vm/src/importlib.rs

```diff
@@ -0,0 +1,1175 @@
+pub const IMPORTLIB_BOOTSTRAP: &str = r#"
```

Comment:
Could we put this in its own file? I don't know if it should be in `vm/src` or somewhere else, but it seems easier to use an `include_str!()`  than to have a 1000 line string literal in a rust source file.

Reply:
Way better. Thanks :)